modules/builds-source-secret-ssh-key-auth.adoc

```diff
@@ -32,5 +32,17 @@ the public key. The private key is used to access your private repository.
 ----
 $ oc create secret generic <secret_name> \
     --from-file=ssh-privatekey=<path/to/ssh/private/key> \
+    --from-file=<path/to/known/hosts> \ <1>
     --type=kubernetes.io/ssh-auth
 ----
+<1> Optional: Adding this field enables strict server host key check.
++
+[WARNING]
+====
+Skipping `known_hosts` file while creating the secret makes the build vulnerable to potential man-in-the-middle (MITM) attack.
+====
++
+[NOTE]
+====
+Ensure the `known_hosts` file includes GitLab key.
```

Comment:
Yeah the literal pull of this from https://bugzilla.redhat.com/show_bug.cgi?id=1745789#c4 is not suitable for documenting from a generic perspective

It should be something like:

Ensure the `known_hosts` files includes an entry for the host where your source code is located.

Reply:
@gabemontero  Ensure the known_hosts file includes an entry for the host of your source code. 
Does this sound technically correct?

Reply:
sure that's fine @snarayan-redhat

Reply:
@gabemontero I have made the suggested change. Kindly approve.